README.Rmd

```diff
@@ -1,72 +1,48 @@
 ---
-title: "d&r update"
+title: "Data @ Reed \nR Resources"
 output: github_document
 ---
 
 ```{r, echo = FALSE, message = FALSE, warning = FALSE}
 library(knitr)
 ```
 
-yall! !!!
+Welcome to the Data at Reed GitHub repository. This contains the source code for the new and improved [Data at Reed](https://www.reed.edu/data-at-reed/resources/R/index.html) R Resources site. The site contains walkthroughs and tutorials for some common R usages. 
 
-this is a GitHub repository that will help us manage an update to the [Data at Reed](data-at-reed) site. specifically, the information at [Analyzing & Visualizing Data > R tutorials]! ideally, we can point people who come into the mLab to that site to help address common issues.
+## [Contents](https://www.reed.edu/data-at-reed/resources/R/index.html)
 
-## what now?
+#### 01. Getting Started with R
+The RStudio Server, Desktop R, Script vs Markdown terminology, `thesisdown`, and meeting the `palmerpenguins`
 
-Leila and I have added all of the previous files in the `old_docs` folder, and an outline of what the new content could look like below. Each header is a single document, and below it are some packages and functionality that we'll discuss what they do, maybe give an example, link out to resources that are maintained by authors and collaborators and _how_ to do it, and probably not much more. Each of the new documents are in `content`, and have corresponding issues linked to the work that happens on them.
+#### 02. Loading Data
+`readr`, `readxl`, `googlesheets4`, and `rvest`
 
-## a new outline
+#### 03. Wrangling Data
+Tidy data principles, `tidyr`, `dplyr`, and `stringr`
 
-### R at Reed
-The RStudio Server & `thesisdown`
+#### 04. Presentation & Visualization
+graphs with `ggplot2`, tables with `knitr` and `kableExtra`
 
-### Getting Started
-Installing R & RStudio, some terminology, and `palmerpenguins`
+#### 05. More Resources
+Learning more & getting connected- cheatsheets, online textbooks, and where to ask for help.
 
-### Loading Data
-`readr`, `readxl`, `googlesheets4`, `rvest`
 
-### Data Wrangling
-`tibble`, `tidyr`, `dplyr`
-
-### Presentation & Visualization
-`ggplot2`, `kable`, RMarkdown
-
-### More Resources
-Learning more & getting connected (links to the free online textbook, our hours in the MLab?)
-
-## keeping-in-mind-things
+## About the Data
 
 We'll be using `palmerpenguins` to demonstrate important functions, instead of `iris`! Here's some [documentation](https://allisonhorst.github.io/palmerpenguins/articles/intro.html) for `palmerpenguins`, and some info on [why we're not using iris](https://www.meganstodel.com/posts/no-to-iris/).
```

Comment:
I know these lines aren't due to your changes, but maybe start this off with

"We use `palmerpenguins` as an example dataset to demonstrate important functions instead of `iris`."

instead? I blame only myself for the previous ambiguity and unnecessary commas.🦥